synapse/replication/tcp/commands.py

```diff
@@ -107,7 +107,7 @@ def to_line(self):
         return " ".join((
             self.stream_name,
             str(self.token) if self.token is not None else "batch",
-            json.dumps(self.row),
+            json.dumps(self.row, namedtuple_as_object=False),
```

Comment:
I think if you're going to rely on stuff that's only in simplejson, you ought to import it as `simplejson` rather than `json`.

Of course, this blows my "simplejson is the same as json" stuff out of the water...